plugin.rb

```diff
@@ -422,9 +422,9 @@ def valid_event
     end
 
     grouped.each do |_, v|
-      v[:name] = v[:name].sort.join(", ")
-      v[:usernames].sort!
-      v[:usernames].uniq!
+      v[:name].sort!.uniq!
```

Comment:
You can chain bang methods?! 😲

Reply:
`sort!` yes, because it always returns itself. `uniq!` no, because it returns `nil` if there were no items removed.

Reply:
Weird to have different behavior. Usually, you can't chain bang methods.